polkadot/node/core/prospective-parachains/src/lib.rs

```diff
@@ -146,12 +146,20 @@ async fn run_iteration<Context>(
 					handle_candidate_seconded(view, para, candidate_hash),
 				ProspectiveParachainsMessage::CandidateBacked(para, candidate_hash) =>
 					handle_candidate_backed(&mut *ctx, view, para, candidate_hash).await?,
-				ProspectiveParachainsMessage::GetBackableCandidate(
+				ProspectiveParachainsMessage::GetBackableCandidates(
 					relay_parent,
 					para,
+					n_cores,
```

Comment:
Is this name justified here? Does prospective parachains need to know about cores in this context? We just want n candidates, what for should be our business.